particles/Dataflow/TextClassifier.arcs

```diff
@@ -0,0 +1,40 @@
+// Text Classifier API Example.
+// From: https://docs.google.com/presentation/d/1i-mTjlyWFSHYxvcJj98fIkUkHSu7ya3COa2wZeiqyi8/edit#slide=id.g5907ddee85_0_126
+
+import './Datastores.arcs'
+
+particle TextClassifier in 'empty.js'
+  in Query {} input
+  in Model model
+  out Result {} result
+
+particle APIResponse in 'empty.js'
+  in Result {} result
+  consume responseSlot
+
+particle ExposeAPI
+  out Query {} query
+  consume root
+    provide replySlot
+  
+  // Ensure that the reply slot is filled with data that is derived only from
+  // the query, or from public data.
+  check replySlot data is from handle query or is public
+
+recipe TextClassifierRecipe
+  slot 'rootslotid-root' as root
+  use PublicModel as model
```

Comment:
Do you have failing examples? E.g. with a Private model here?

Reply:
The idea was to have these examples all be passing, and for readers/demo-ees to manually edit them to see how they fail (which kind of represents how it might help catch accidental mistakes made to the recipe later on).